BlueprintUI/Sources/Element/ElementContent.swift

```diff
@@ -413,6 +415,7 @@ private struct LazyStorage: ContentStorage {
         let node = LayoutResultNode(
             element: child,
             layoutAttributes: childAttributes,
+            environment: environment,
```

Comment:
Oh smart! I guess adding this to the layout result does get us the right environment; nice!!

Reply:
Less "smart" and more me just adding the environment to the `backingViewDescription` and finding myself here, where there was already an environment passed in 😂 but yeah, glad it worked it out like that!